src/02-room-details.js

```diff
@@ -49,7 +130,60 @@ function getRoomByDinosaurName(dinosaurs, rooms, dinosaurName) {}
       "Kit Hopkins Education Wing"
     ]
  */
-function getConnectedRoomNamesById(rooms, id) {}
+function getConnectedRoomNamesById(rooms, id) {
+  // // declare 'arrOfRooms' and assign an empty array
+  // const arrOfRooms = [];
+  // // declare 'connectsId' and assign an empty array
+  // let connectsId = [];
+
+  // // iterate through each 'room' in 'rooms'
+  // for (const room of rooms) {
+  //   // if 'room.roomId' matches 'id'
+  //   if (room.roomId === id) {
+  //     // 'connectsId' is reassigned to 'room.connectsTo'
+  //     connectsId = room.connectsTo;
+  //   }
+  // }
+
+  // // edge case #1
+  // if (connectsId.length === 0) {
+  //   return `Room with ID of '${id}' could not be found.`;
+  // }
+  
+  // // iterate through each 'room' in 'rooms'
+  // for (const room of rooms) {
+  //   // if 'connectsId' includes 'room.roomId'
+  //   if (connectsId.includes(room.roomId)) {
+  //     // push 'room.name' into 'arrOfRooms'
+  //     arrOfRooms.push(room.name);
+  //   }
+  // }
+  // // return modified 'arrOfRooms'
+  // return arrOfRooms;
+
+
+  let connectedNames = [];
+
+  for (const room of rooms) {
+    for (const connect of room.connectsTo) {
+      if (id === connect && connect !== 'incorrect-id') {
+        connectedNames.push(room.name);
+      } else if (connect === 'incorrect-id') {
```

Comment:
This is hard coded. Incorrect-id could be anything, it could be jdfhbvjfd4 and your code should be able to detect that its wrong. Knowing that how would you reconsider your code?